src/win/winapi.c

```diff
@@ -156,4 +160,10 @@ void uv_winapi_init(void) {
       GetProcAddress(powrprof_module, "PowerRegisterSuspendResumeNotification");
   }
 
+  user32_module = LoadLibraryA("user32.dll");
```

Comment:
More a question than a comment but do we ever call FreeLibrary on these DLLs? I guess we just expect them to be loaded for the duration of the process? I imagine there are some scenarios where we'd want some of these DLLs to be unloaded earlier if possible but certainly not in this case